networking/p2p/src/main/java/tech/pegasys/teku/networking/p2p/discovery/DiscoveryNetwork.java

```diff
@@ -137,6 +139,11 @@ public void setSyncCommitteeSubnetSubscriptions(Iterable<Integer> subnetIds) {
             .sszSerialize());
   }
 
+  public void setDASCustodySubnetCount(int count) {
```

Comment:
let's add extra into method name so it's clear why it's not set always.
Field name doesn't have anything like `extra` so at least something should highlight this.